infrastructure/templates/public-api/main.bicep

```diff
@@ -238,6 +227,14 @@ module apiContainerAppModule 'components/containerApp.bicep' = if (psqlDbUsersAd
         name: 'ContentApi__Url'
         value: publicUrls!.contentApi
       }
+      {
+        name: 'MiniProfiler__Enabled'
+        value: 'true'
+      }
+      {
+        name: 'ParquetFiles__BasePath'
+        value: 'data/public-api-parquet'
```

Comment:
I think we need a new ticket creating on configuring this Azure File Share at some point, in order to mount the data directory.

Reply:
I've created https://dfedigital.atlassian.net/browse/EES-5127 to cover this.